app/viewmodels/journeys/taskList/NationalInsuranceContributionsViewModel.scala

```diff
@@ -26,22 +28,28 @@ import viewmodels.journeys.{SummaryListCYA, getJourneyStatus}
 
 object NationalInsuranceContributionsViewModel {
 
-  def buildSummaryList(nationalInsuranceStatuses: List[JourneyNameAndStatus])(implicit messages: Messages): SummaryList = {
+  def buildSummaryList(
+      nationalInsuranceStatuses: List[JourneyNameAndStatus])(implicit messages: Messages, taxYear: TaxYear, businessId: BusinessId): SummaryList = {
```

Comment:
taxYear doesn't need to be implicit, and we don't need businessId (you might need to remove it from anywhere it businessId is used for that page in the controller and the routes)

Reply:
Done :)